### PR TITLE
Enhancement: Enable `switch_case_space` fixer

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] enables the `switch_case_space` fixer
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/doc/rules/control_structure/switch_case_space.rst.

### CODE REVIEW

File: error.php [hunk 1]
```diff
@@ -122,9 +122,9 @@
 // Some nice URLs for getting something for download
 if (preg_match("!^get/([^/]+)$!", $URI, $what)) {
     switch ($what[1]) {
-        case "php"           : $URI = "downloads"; break;
-        case "docs"          : // intentional
-        case "documentation" : $URI = "download-docs"; break;
+        case "php": $URI = "downloads"; break;
+        case "docs": // intentional
+        case "documentation": $URI = "download-docs"; break;
```
Comment (cmb69) [inline error.php:127]: I'd reformat manually right away:
```suggestion
        case "php":
            $URI = "downloads";
            break;
        case "docs":
        case "documentation":
            $URI = "download-docs";
            break;
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Adding the switch_case_space fixer looks reasonable to me, but I'd reformat one of the switches (partially to avoid discussion of whether it is better to align the colons).

**Review by kamil-tekiela — APPROVED** [review]

**Comment by localheinz** [comment]
@cmb69 @kamil-tekiela 

Is this good to go?

**Comment by cmb69** [comment]
> Is this good to go?

Yes. Thank you!

**Pull request merged**